lua/nvim-tree/colors.lua

```diff
@@ -45,7 +45,9 @@ local function get_hl_groups()
     GitStaged = { fg = colors.green },
     GitMerge = { fg = colors.orange },
     GitRenamed = { fg = colors.purple },
-    GitNew = { fg = colors.yellow }
+    GitNew = { fg = colors.yellow },
+
+    WindowPicker = { gui = "bold", fg = "#ededed", bg = "#3d84b4" }
```

Comment:
I don't think we should. We want the window picker to stick out so that it grabs the user's attention. For reference, coc-explorer uses a static color for this, and it never felt out place for me, regardless of color scheme.

I tried linking this to a number of different highlight groups, and I couldn't get it to look good and also stick out enough to grab the user's attention across multiple color schemes.

This white on blue is quite neutral, and it looks quite decent with all the color schemes I tried. I think it's a good default.

Reply:
I understand not using one of the primary theme colors, or an existing highlight group. But perhaps you could create a new highlight group so that theme developers can set the colors if they decide to add custom highlights for this extension.